### PR TITLE
[13.0][FIX] sale_order_move_menu_spil: missing dependencies

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Add invoice_line_ids.price_unit and quantity_done dependencies, update all stock move [('sale_line_id', '!=', False), ('state', '=', 'done')] with post-migration system

Need to install it in test, I will keep the status informed...

### CODE REVIEW

File: sale_order_move_menu_spil/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "AGPL-3",
-    "version": "13.0.2.0.0",
+    "version": "13.0.2.1.0",
```
Comment (dalonsod) [inline sale_order_move_menu_spil/__manifest__.py]: This is a fix, so
```suggestion
    "version": "13.0.2.0.1",
```

File: sale_order_move_menu_spil/migrations/13.0.2.1.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+def migrate(cr, version):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        sale_move_ids = env['stock.move'].search([
+            ('sale_line_id', '!=', False),
+            ('state', '=', 'done')
+        ])
+        for record in sale_move_ids:
+            record._compute_sale_move_menu()
```
Comment (dalonsod) [inline sale_order_move_menu_spil/migrations/13.0.2.1.0/post-migration.py]: Compute method is already multi, so you can simply use
```suggestion
        sale_move_ids._compute_sale_move_menu()            
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments (minor changes)

**Comment by ChristianSantamaria** [comment]
Done and tested 👍

**Review by dalonsod — APPROVED** [review]

**Pull request merged**